### PR TITLE
build: adds scss and postcss support in build pipeline

### PR DESCRIPTION
**Pull request by timbru31**
Depends on #3

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
-	eleventyConfig.setTemplateFormats(["liquid"]);
+const autoprefixer = require("autoprefixer");
+const cssnano = require("cssnano");
+const { default: postcss } = require("postcss");
+const processSassFiles = require("./config/process-sass");
+
+module.exports = (eleventyConfig) => {
+	processSassFiles("./styles/index.scss", "./_includes/css/main.css");
+
+	eleventyConfig.setTemplateFormats(["liquid", "njk"]);
```
Comment (tam-kis) [inline .eleventy.js:9]: Is this only possible with nunjucks?

Comment (timbru31) [reply .eleventy.js:9]: Yes, see the little comment in `pages/stylesheet.njk`, using Liquid I've got errors when compiling.

Comment (tam-kis) [reply .eleventy.js:9]: Ah I see, sorry

Comment (timbru31) [reply .eleventy.js:9]: No problem!

**Review by tam-kis — APPROVED** [review]

**Pull request merged**